### PR TITLE
docs: update list of available crates

### PR DESCRIPTION
**Pull request by sripwoud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,20 +64,21 @@
...
-                    <img src="https://img.shields.io/crates/d/example?style=flat-square" alt="Crate version" />
+                <a href="https://privacy-scaling-explorations.github.io/zk-kit.rust/imt">
+                    <img src="https://img.shields.io/crates/v/zk-kit-rs-imt?style=flat-square" alt="Crate version" />
                 </a>
             </td>
             <td>
-                <a href="https://docs.rs/example">
-                    <img src="https://img.shields.io/crates/v/example?style=flat-square" alt="Crate downloads" />
+                <a href="https://privacy-scaling-explorations.github.io/zk-kit.rust/imt">
+                    <img src="https://img.shields.io/crates/d/zk-kit-rs-imt?style=flat-square" alt="Crate downloads" />
```
Comment (cedoor) [inline README.md:81]: @sripwoud is there anything similar to NPM scopes in Rust?

I don't think we need to specify "rs" 🤔

Comment (sripwoud) [reply README.md:81]: on crates.io you can't have a scope/organization (`@scope`) in the pkg name like in npm.
But regarding ownership, it is possible to add a github team 
with `cargo owner --add github:org:team`
see [docs](https://doc.rust-lang.org/cargo/reference/publishing.html?highlight=orga#cargo-owner)

So should we create a zk-kit team on github?

also captured this issue to add a workflow to release the crates #10

Comment (sripwoud) [reply README.md:81]: about the pkg name, should we go with `zk-kit-imt`?
And name all our crates `zk-kit-<pkg>`?

Comment (sripwoud) [reply README.md]: renamed to `zk-kit-imt`

Comment (cedoor) [reply README.md]: Yes makes sense!

**Review by cedoor — APPROVED** [review]

**Pull request merged**